src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-keyvault-secret/diagnostics.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
I'm wondering if it might be worth switching to use the output formatter we use for some of the Decompiler tests (which inlines warnings/errors into a Bicep file), so that reviewers of snippets would get a visual indicator of what the snippet + replacement template looks like (to make reviewing external contributions simpler. For example:

https://github.com/Azure/bicep/blob/d9ba1b26036251aca1b8f086c2a3aaa9a9ac6fbe/src/Bicep.Decompiler.IntegrationTests/Working/conditional/main.bicep#L53

Reply:
I pushed an example implementation of this here: https://github.com/Azure/bicep/commit/a37a3215a3af9a11f867b3b5f0f6fec7cb50ce6c

The `main.combined.bicep` file is the result of the snippet + placeholder files. Diagnostics are also included in the combined file (e.g. [here](https://github.com/Azure/bicep/blob/a37a3215a3af9a11f867b3b5f0f6fec7cb50ce6c/src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/module/main.combined.bicep#L2)) - making it easy to review external contributions.

Reply:
#2247

Reply:
Thanks Anthony for the example. It looks awesome. I really like the combined file. I looked at the changes and it looks good as is, and I updated it here- https://github.com/Azure/bicep/commit/2a82775dc6fa1d4c2b78ff052bde0153ced945db. I made one minor change to not have any errors for module snippet. here- https://github.com/Azure/bicep/commit/2aa6c1a1dee867715654f29363a6423b2392f14b
Let me know if you have  any concerns/questions.

Reply:
Looks great!